packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.4.0
+
+* Adds option to configure HTTP headers via `VideoPlayerController` to fix access to M3U8 files on Android.
+* Adds option to configure the User-Agent in the HTTP headers.
```

Comment:
I missed this before, but it sounds like this is describing new API (which doesn't exist) so it's not at all clear to a reader how to do this. How about: "Allows setting the ExoPlayer user agent by passing a `User-Agent` HTTP header."